linode/manager
# refactor: [M3-6488] - React Query for Linode Details - General Details

## Description 📝
- Switches the "top half" of the Linodes details page to use React Query
- Once we make more progress, another "general" PR like this one will probably be needed to address the comments I left
- The code might still looks and feel a bit crusty but we'll be able to get things cleaner as we migrate more components

## Preview 📷
 > **Note**: There should be no visual changes
 
![Screenshot 2023-05-08 at 6 19 19 PM](https://user-images.githubusercontent.com/115251059/236950403-04bd2406-c5ca-44f9-93c0-633caf39ca6c.jpg)

## How to test 🧪
- Verify that the linodes details page is still working as expected
- Verify e2es pass
